jdi-light-mobile-tests/src/main/java/io/github/epam/custom/Result.java

```diff
@@ -3,5 +3,6 @@
 import com.epam.jdi.tools.DataClass;
 
 public class Result extends DataClass<Result> {
-    public String name, description, link;
+    public String name;
+    public String description, link;
```

Comment:
Codacy found an issue: [Use one line for each declaration, it enhances code readability.](https://app.codacy.com/gh/jdi-testing/jdi-light/pullRequest?prid=6343884)